tests/check_consistent.py

```diff
@@ -0,0 +1,29 @@
+import os
```

Comment:
Can you add documentation here that says that we can't use symlinks for technical reasons, so we use this as a way to make sure certain groups of files stay identical?